binaries/scoot-snapshot-db/main.go

```diff
@@ -53,6 +56,11 @@ func (i *injector) RegisterFlags(rootCmd *cobra.Command) {
 }
 
 func (i *injector) Inject() (snapshot.DB, error) {
+	dbTempDir, err := ioutil.TempDir("", "")
+	if err != nil {
+		return nil, err
+	}
+
```

Comment:
not part of this PR, but could we remove the comment below (or at least the parts about talking to the daemon)